### PR TITLE
18F Delivery Content

### PR DESCRIPTION
**Pull request by emileighoutlaw**
18F Delivery page content. @elainekamlley: bullet points are still being obstinate.

### CODE REVIEW

**Comment by gboone** [comment]
What's wrong with the bullet points?

**Comment by elainekamlley** [comment]
AH! We don't know! They also don't show up on `/consulting`

**Comment by gboone** [comment]
#937 will bring a fix into this PR. It's a CSS error; apparently we added a list style for blog posts but not for the pages template.

File: pages/delivery.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- [A new look at the Freedom of Information Act](https://18f.gsa.gov/2014/09/04/a-new-look-at-the-freedom-of-information-act/)
+- [Creating an OpenFEC](https://18f.gsa.gov/2014/08/21/creating-an-open-fec/)
+- [A few notes on NotAlone.gov](https://18f.gsa.gov/2014/05/09/a-few-notes-on-notalone-gov/)
+
+
+
+<!-- Obfuscate our email -->
+<div>
+  <script>
+    var parts = ["18FC", "@", "gsa", ".gov"];
```
Comment (gboone) [inline pages/delivery.md]: @emileighoutlaw change the "18FC" to "inquiries" or whatever the part of the email address before the @ is supposed to be.

File: pages/delivery.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Deadlines can change. Needs can change. We make our teams flexible enough to always deliver something high quality and high value. Developing, standing up, and sharing with users a minimally viable product — one that isn’t fully functional — is a key component of what we do. This allows us to gather feedback and make edits before a product is finalized.
+
+## Design centric and data driven
+User experience and usability research is embedded into our projects from the start. We want to build the right thing, not just anything. Accomplishing this means speaking to users regularly and having them test our work as often as possible. 
+
+Our developers, designers, and project managers work side-by-side with researchers to understand the impact of our design decisions. Our product teams care about usability and are often found alongside user-experience researchers testing versions of our products to understand how people interact with them.
+
+##Contact us
+If you work at an agency and are interested in partnering with 18F, send us an email at <a href="mailto:inquiries@18f.gsa.gov" itemprop="email"><i title= "email 18F"></i> inquiries@18f.gsa.gov</a>.
```
Comment (gboone) [inline pages/delivery.md]: These two instances of inquiries@18f.gsa.gov should be inquiries18F@gsa.gov

**Comment by elainekamlley** [comment]
Nice work @emileighoutlaw! Thanks for the support @gboone!

**Pull request merged**